M2/Macaulay2/packages/undistributed-packages/ExampleIdeals.m2

```diff
@@ -8,7 +8,9 @@ newPackage(
 		  Email => "mike@math.cornell.edu", 
 		  HomePage => "http://www.math.cornell.edu/~mike/"}},
```

Comment:
The changes to this file seem unrelated to saturation and ideal quotients.  Were they intended?

Reply:
The changes come from commit 1e51917be88e97686a0567798b512ea68eebc41b by @mikestillman. The commit only includes that file and the commit message describes the change correctly, so it is not an accidental inclusion, but I don't know if it was intended for this branch or not.

Reply:
This was somewhat unintended.  I needed to update the code (include the newer package GraphicalModels, rather than the older Markov) in order to get older code for generating examples to work.  So perhaps the change should go elsewhere, but it was needed for examples.